.github/archlinux/build

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```

Comment:
First things first: I am very sorry for the late response.

Shell scripts should have the file ending `.sh`.